src/algebra/group/pi.lean

```diff
@@ -172,22 +172,29 @@ open pi
 
 variables (f)
 
-/-- The zero-preserving homomorphism including a single value
+/-- The one-preserving homomorphism including a single value
 into a dependent family of values, as functions supported at a point.
 
-This is the `zero_hom` version of `pi.single`. -/
-@[simps] def zero_hom.single [Π i, has_zero $ f i] (i : I) : zero_hom (f i) (Π i, f i) :=
-{ to_fun := single i,
-  map_zero' := single_zero i }
+This is the `one_hom` version of `pi.mul_single`. -/
+@[to_additive zero_hom.single "The zero-preserving homomorphism including a single value
```

Comment:
You can teach `to_additive` to rename `mul_single` to `single` (instead of `add_single`), by inserting the following as `algebra/group/to_additive.lean` line 206:
```lean
| is_comm ("mul" :: "single" :: s)    := add_comm_prefix is_comm "single"    :: tr ff s
```
After you apply the change, `to_additive` will helpfully point out exactly the names it correctly autogenerates. So we don't need to name everything twice :D